docs/assistant/security-assistant.asciidoc

```diff
@@ -185,6 +185,8 @@ The **Knowledge base** tab of the AI Assistant settings menu allows you to enabl
 [[rag-for-esql]]
 ==== Knowledge base for {esql}
 
+NOTE: {esql} features are turned on by default and are controlled by the {kibana-ref}/advanced-options.html#kibana-general-settings[`enableESQL`] advanced setting.
```

Comment:
```suggestion
NOTE: {esql} is enabled by default in {kib}. It can be
disabled using the `enableESQL` setting from the
{kibana-ref}/advanced-options.html[Advanced Settings].
+
This will hide the {esql} user interface from various applications.
However, users will be able to access existing {esql} artifacts like saved searches and visualizations.
```